adagrams/game.py

```diff
@@ -1,13 +1,122 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+        'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+        'D': 2, 'G': 2,
+        'B': 3, 'C': 3, 'M': 3, 'P': 3,
+        'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+        'K': 5,
+        'J': 8, 'X': 8,
+        'Q': 10, 'Z': 10
+}
+
+
 def draw_letters():
-    pass
+    letter_pool_list = []
+
+    for letter in LETTER_POOL:
+        count = LETTER_POOL[letter]
+        for copy in range(count):
+            letter_pool_list.append(letter)
```

Comment:
This works, but it would be a little more straight forward to use the dictionary method `items()` instead, like:
```suggestion
    for letter, frequency in LETTER_POOL.items(): 
        for _ in range(frequency): 
            letter_pool_list.append(letter)
```

Reply:
Prefer lines 46-49 be written in a helper function like `get_letter_pool` and then you'd just invoke that function here in `draw_letters`. 

Doing so would do a couple things: 
- make `draw_letters` more concise
- make `draw_letters` concerned with less logic
-  makes your code more self-documenting because instead of reading through lines of logic and surmising what it does, I would just be reading a function call that is very descriptive. 

So line 44 would like this after refactoring
```py
    letter_pool_list = get_letter_pool()